src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './GenreSelect.scss'
+
+interface Props {
+  genres: string[];
+  initialSelectedGenre: string;
+  onSelect: (text: string) => void;
+}
+
+function GenreSelect({genres, initialSelectedGenre, onSelect}: Props) {
+  const [selectedGenre, setSelectedGenre] = useState(initialSelectedGenre);
+
+  const handleGenreSelect = (genre: string) => {
+    setSelectedGenre(genre);
+    onSelect(genre);
+  }
+
+  return (
+    <div className="genre-select m-3">
+      {genres.map((genre) => (
+          <button
+              key={genre}
+              onClick={() => handleGenreSelect(genre)}
+              className={' ' + (genre === selectedGenre && 'active')}>
```

Comment:
```suggestion
              className={genre === selectedGenre ? 'active' : ''}>
```
Or use template string, otherwise you can see classes like ` false`.

Reply:
Rewrote to a suggested version